### PR TITLE
Update dm-webui-guide.md

### PR DESCRIPTION
**Pull request by sunzhaoyang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Remove dashboard introduction.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dm/dm-webui-guide.md [hunk 1]
```diff
@@ -16,7 +16,6 @@ DM WebUI 是一个 TiDB Data Migration (DM) 迁移任务管理界面，方便用
 
 DM WebUI 主要包含以下界面：
 
-- **Dashboard**：包含了 DM 中同步任务的主要监控信息和运行状态信息，帮助用户快速了解任务的整体运行状况，以及与延迟、性能相关的重要信息。
```
Comment (WizardXiao) [inline dm/dm-webui-guide.md:19]: Maybe the picture `/media/dm/dm-webui-preview-cn.png` needs to remove `Dashboard` too.

File: dm/dm-webui-guide.md [hunk 2]
```diff
@@ -34,56 +33,6 @@ DM WebUI 主要包含以下界面：
 
 ## 使用前提
```
Comment (ran-huang) [inline dm/dm-webui-guide.md:34]: 这一行也删了吧？

File: dm/dm-webui-guide.md [hunk 3]
```diff
@@ -32,58 +31,6 @@ DM WebUI 主要包含以下界面：
...
-        allow_embedding = true
-        ```
-
-    2. 执行 `tiup dm reload` 使新的配置生效。
-
-## Dashboard
-
-要查看迁移任务的监控，你可访问 **Dashboard** 页面。**Dashboard** 是内嵌了 DM 的 Grafana Dashboard，包含 `Standard` 和 `Professional` 两个视图，分别从标准角度展现监控信息，和从更专业的角度展现更详细的监控信息。
-
 ## 数据迁移
```
Comment (ran-huang) [inline dm/dm-webui-guide.md:34]: **数据迁移**是二级标题，**上游配置、任务列表、同步详情**是三级标题吧，麻烦调整一下。

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

**Review by ran-huang — APPROVED** [review]

**Review by WizardXiao — APPROVED** [review]

**Pull request merged**